lib/stack_master/sparkle_formation/compile_time/min_length_validator.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'value_validator'
+
+module StackMaster
+  module SparkleFormation
+    module CompileTime
+      class MinLengthValidator < ValueValidator
+
+        KEY = :min_length
+
+        def initialize(name, definition, parameter)
+          @name = name
+          @definition = definition
+          @parameter = parameter
+        end
+
+        private
+
+        def check_is_valid
+          return true unless @definition[:type] == :string
+          return true unless @definition.key?(KEY)
+          invalid_values.empty?
+        end
+
+        def invalid_values
+          values = build_values(@definition, @parameter)
+          values.select {|value| value.length < @definition[KEY].to_i}
```

Comment:
As above.

Reply:
Might I suggest a linking tool, if a particular format is required. This is just Rubymine default formatting

Reply:
I'll let @shivaman do the formatting.